recipe/meta.yaml

```diff
@@ -68,18 +73,22 @@ outputs:
         - test -f ${PREFIX}/lib/libLLVMSPIRVLib.so.{{ abi_version }}     # [linux]
 
   - name: libllvmspirv
-    script: install.sh
+    script: install.sh   # [not win]
+    script: install.bat  # [win]
     build:
       run_exports:
         - {{ pin_subpackage("libllvmspirv" ~ major) }}
       ignore_run_exports_from:
         - zlib
-        - spirv-tools
+        - spirv-tools  # [not win]
+      script_env:
+        - MAJOR_VERSION={{ major }}  # [win]
     files:
-      - lib/libLLVMSPIRVLib.dylib       # [osx]
-      - lib/libLLVMSPIRVLib.so          # [linux]
-      - lib/pkgconfig/LLVMSPIRVLib.pc
-      - include/LLVMSPIRVLib/
+      - lib/libLLVMSPIRVLib.dylib          # [osx]
+      - lib/libLLVMSPIRVLib.so             # [linux]
+      - {{ library }}lib/LLVMSPIRVLib.lib  # [win]
```

Comment:
There's no DLL?

Reply:
@isuruf unfortunately, no. LLVM does not support shared libraries on windows:
https://www.llvm.org/docs/CMake.html#llvm-related-variables
https://github.com/KhronosGroup/SPIRV-LLVM-Translator/blob/main/lib/SPIRV/CMakeLists.txt#L43

Reply:
BTW, if you know any workarounds - I would appreciate. I've tried `clangxx` - it did not work. The only other option could be to use `m2gw64`, but it will bring it's own runtime.

Reply:
`.lib` is only dev package dependency (`libllvmspirv`) and is not presented at the `llvm-spirv` packages.